tests/pytests/test_stats.py

```diff
@@ -42,26 +43,25 @@ def runTestWithSeed(env, s=None):
     env.expect('ft.config set FORK_GC_CLEAN_THRESHOLD 0').ok()
 
     env.expect('FT.CREATE idx SCHEMA n NUMERIC').ok()
-    check_empty(env, idx)
+    check_empty(env, idx, 0)
 
     for i in range(count):
         conn.execute_command('HSET', 'doc%d' % i, 'n', i)
 
     env.expect('FT.SEARCH idx * LIMIT 0 0').equal([count])
     for i in range(count):
         env.expect('FT.SEARCH', 'idx', '@n:[%d,%d]' % (i, i))#.equal([1, 'doc%d' % i, ['n', str(i)]])
-    #check_not_empty(env, idx)
+    check_not_empty(env, idx)
```

Comment:
The consumed memory was checked in the first test which creates documents with increasing integer values, for the tests that use random float or random integer, it is not possible to know the exact memory consumed, because it depends on the distribution of the range nodes in the range node tree.